.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
-- '2.6'
 - '2.7'
-- '3.2'
```

Comment:
Got it. In that case, should we remove the version from this yaml altogether and let travis use the default one?